filament/backend/src/opengl/gl_headers.h

```diff
@@ -46,12 +66,22 @@
 
 #endif
 
+/* Validate the header configurations we aim to support */
 
-#if (!defined(GL_ES_VERSION_2_0) && !defined(GL_VERSION_4_1))
-#error "Minimum header version must be OpenGL ES 2.0 or OpenGL 4.1"
+#if defined(GL_VERSION_4_5)
+#elif defined(GL_ES_VERSION_3_1)
+#elif defined(GL_ES_VERSION_3_0)
+#   if !defined(IOS) && !defined(__EMSCRIPTEN__)
+#       error "GLES 3.0 headers only supported on iOS"
```

Comment:
Should this error read "GLES 3.0 headers only supported on iOS and WebGL2"?

Reply:
yep. thanks.